lib/interpret_date/version.rb

```diff
@@ -1,3 +1,3 @@
 module InterpretDate
-  VERSION = "1.3.5"
+  VERSION = "1.4.0"
```

Comment:
Just a stickler here, but since you made no breaking changes whatsoever, this should be a patch version.